tests/common/snappi_tests/traffic_generation.py

```diff
@@ -611,9 +612,15 @@ def verify_pause_frame_count_dut(rx_dut,
                 pytest_assert(pfc_pause_rx_frames == 0,
                               "PFC pause frames with no bit set in the class enable vector should be dropped")
             else:
-                pytest_assert(pfc_pause_rx_frames > 0,
-                              "PFC pause frames should be received and counted in RX PFC counters for priority {}"
-                              .format(prio))
+                if "Cisco" in tx_dut.facts['hwsku'] and prio not in lossless_prios:
+                    pytest_assert(
+                        pfc_pause_rx_frames == 0,
```

Comment:
Could you check the pfc rx counter for lossless only? Remove pfc rx counter check for lossy.